R/rbmi.R

```diff
@@ -1,2 +1,21 @@
+
+#' @description
+#'
+#' The rbmi package is used to perform reference based multiple imputation. The package
+#' provides implementations for common imputation strategies whilst allowing the user to
+#' select between various standard Bayesian and Frequentest approaches.
+#'
+#' The package is designed around its 4 core functions:
+#' - [draws()] - Fits multiple imputation models
+#' - [impute()] - Imputes multiple datasets
+#' - [analyse()] - Analyses multiple datasets
+#' - [pool()] - Pools multiple results into a single statistic
+#'
+#' To learn more about rbmi, please see the quickstart vignette:
+#'
+#' `vignette(topic= "quickstart", package = "rbmi")`
+#'
```

Comment:
Does it make sense to link here and/or in the README file to the statistical specifications document that we plan to finalize (when this will be ready)?

Reply:
When its ready yes, but we can't link to it yet :)

Reply:
sure. I will accept the PR then!